src/c/vernier_c.cpp

```diff
@@ -19,9 +19,10 @@
 #include "vernier_get_wtime.h"
 #include "vernier_mpi.h"
 #include <cstring>
+#include <string>
 
 extern "C" {
-void c_vernier_init(const MPI_Fint *const client_comm_handle);
+void c_vernier_init(MPI_Fint *const client_comm_handle, char *const tag);
```

Comment:
Const shouldn't be removed here and I think you should be able to have one on the `char` pointer as well
```suggestion
void c_vernier_init(const MPI_Fint *const client_comm_handle, const char *const tag);
```